library/bigip_selfip.py

```diff
@@ -526,11 +528,21 @@ def create(self):
             )
         else:
             vlan = "/%s/%s" % (partition, vlan)
-
         try:
-            ipin = "%s/%s" % (address, netmask)
-            ipnet = IPNetwork(ipin)
-            params['address'] = "%s/%s" % (ipnet.ip, ipnet.prefixlen)
+            if address.find('%') != -1:
```

Comment:
This if statement is not required now, side effect of a plan change :) 

Deleting now...